### PR TITLE
copy headers to temp_inc to accommodate to devel branch

### PR DESCRIPTION
**Pull request by microSharjeel**

### CODE REVIEW

**Comment by microSharjeel** [comment]
Hi,
There is no option of sp ram there in sram.v now 
earlier there was a choice between dp and sp rams which could have been chosen with USE_SPRAM variable.
why sp ram is not there now? It gives error

File: hardware/asic/skywater/Makefile [hunk 1]
```diff
@@ -24,19 +23,21 @@ sram: openram_config.py
...
+	cp ../../../submodules/MEM/hardware/rom/iob_rom_sp/iob_rom_sp.v temp_src
 	cp boot.hex temp_src
 	sed -i "s/m4/met4/g" $(wildcard sram/*.lef)
 	sed -i "s/m3/met3/g" $(wildcard sram/*.lef)
 
 DEL:   TEMP
 	sed -i.bak -e '7d' temp_src/system.v
 ADD:   DEL
-	sed -i "5i \`include \"iob_uart.vh\"" temp_src/system.v
 	sed -i "1i \`include \"iob-soc-defs.vh\"" temp_src/system.v
```
Comment (jjts) [inline hardware/asic/skywater/Makefile:41]: where is iob-soc-defs.v ?

Comment (microSharjeel) [reply hardware/asic/skywater/Makefile:41]: iob-soc-def.v is generated and copied to temp_inc in the Makefile kindly check code for Makefile

File: hardware/asic/skywater/Makefile [hunk 2]
```diff
@@ -24,19 +23,21 @@ sram: openram_config.py
 
 iob_sp_ram_be.v:
 	$(MEM_DIR)/software/python/memakerwrap.py $(ASIC_MEM_TECH) iob_sp_ram_be $(ASIC_SPRAM_TYPE) 0 1 $(SRAM_W) 8 4 1 > $@
-TEMP: boot.hex
+TEMP: sw boot.hex
 	$(MKDIR) temp_src
 	$(MKDIR) temp_inc
 	cp $(VSRC) temp_src
 	cp $(VHDR) temp_inc
+	cp ../../../submodules/LIB/hardware/include/iob_s_if.vh temp_inc
```
[the remark this answers is not in the document]
Comment (microSharjeel) [reply hardware/asic/skywater/Makefile:31]: ok 
i will add them

**Pull request closed without merging**